qemu/tests/vsock_hotplug.py

```diff
@@ -48,27 +49,29 @@ def run(test, params, env):
                           'dmesg output.', logging.info)
     addr_pattern = params['addr_pattern']
     device_pattern = params['device_pattern']
-    lspci_cmd = 'lspci'
+    check_vsock_cmd = params.get('lspci_cmd', 'lspci')
```

Comment:
`check_vsock_cmd` & `lspci_cmd`, keep the namespace consistent.

Reply:
solved